integreat_cms/cms/forms/push_notifications/push_notification_form.py

```diff
@@ -7,6 +7,21 @@ class PushNotificationForm(CustomModelForm):
     Form for creating and modifying push notification objects
     """
 
+    def __init__(self, **kwargs):
+        r"""
+        Initialize push notification form
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+        """
+        # Instantiate CustomModelForm
+        super().__init__(**kwargs)
+
+        # Make fields disabled when push notification was already sent
+        if self.instance.sent_date:
+            self.fields["channel"].disabled = True
+            self.fields["mode"].disabled = True
```

Comment:
If the mode is changed to "only available" after sending, the messages are contained in the API response, but empty:
```
$ curl -s http://localhost:8000/api/testumgebung/fa/fcm/ | python -m json.tool

[
    {
        "id": "8",
        "title": "",
        "message": "",
        "timestamp": "2022-10-14T13:30:34.942Z",
        "last_updated": "2022-10-14T15:30:34.942+02:00",
        "channel": "news"
    }
]
```

So I suggest to disable these fields after a notification was sent.